src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/DemoAccountServiceImpl.java

```diff
@@ -12,20 +13,20 @@
     public static final UUID ACCOUNT_WITH_EFILING_ROLE = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fa");
     public static final UUID ACCOUNT_WITHOUT_EFILING_ROLE = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fb");
 
-    private Map<String, CsoAccountDetails> csoAccounts = new HashMap<>();
+    private Map<String, AccountDetails> csoAccounts = new HashMap<>();
 
     public DemoAccountServiceImpl() {
 
-        CsoAccountDetails accountWithEfilingRole = new CsoAccountDetails(BigDecimal.TEN, BigDecimal.TEN, true);
-        CsoAccountDetails accountWithOutEfilingRole = new CsoAccountDetails(BigDecimal.TEN, BigDecimal.TEN, false);
+        AccountDetails accountWithEfilingRole = new AccountDetails(BigDecimal.TEN, BigDecimal.TEN, true, "", "", "", "");
+        AccountDetails accountWithOutEfilingRole = new AccountDetails(BigDecimal.TEN, BigDecimal.TEN, false, "", "", "", "");
```

Comment:
```suggestion
        AccountDetails accountWithoutEfilingRole = new AccountDetails(BigDecimal.TEN, BigDecimal.TEN, false, "", "", "", "");
```

Reply:
avoid inline suggestion for code, this would have break the unit test, fixed though!